This patch was originated in-house.  It is to work around a bug (21825650)
in getrandom(2).  That bug was fixed in S12, and in an 11.3 SRU, but not in
time for 11.3 GA, so this patch is still needed for FOSS Evaluation builds.

diff -wpruN '--exclude=*.orig' a~/Python/random.c a/Python/random.c
--- a~/Python/random.c	1970-01-01 00:00:00
+++ a/Python/random.c	1970-01-01 00:00:00
@@ -154,7 +154,7 @@ py_getrandom(void *buffer, Py_ssize_t si
             /* ENOSYS: the syscall is not supported by the kernel.
                EPERM: the syscall is blocked by a security policy (ex: SECCOMP)
                or something else. */
-            if (errno == ENOSYS || errno == EPERM) {
+            if (errno == ENOSYS || errno == EINVAL || errno == EPERM) {
                 getrandom_works = 0;
                 return 0;
             }
